src/pages/home/sidebar/ChatSwitcherView.js

```diff
@@ -103,6 +101,10 @@ class ChatSwitcherView extends React.Component {
             ChatSwitcher.show();
             this.textInput.focus();
         }, ['meta'], true);
+
+        if (this.props.isChatSwitcherActive) {
+            this.updateSearch(this.state.search);
+        }
```

Comment:
Why this change?

Reply:
`updateSearch` function sets state `options` as props of the `ChatSwitcherList`.
If options is empty, `ChatSwitcherList` is not rendered.
`updateSearch` was fired when focus on search input before.
Now, it should be called when `ChatSwitcherView` component mounted, because this component receive `isChatSwitcherActive: true` when clicking magnifying glass icon.

Reply:
Alright. But why continue to check for `isChatSwitcherActive` it's _always_ active now isn't it? Looking at the code we won't render the `ChatSwitcherView` unless `isChatSwitcherActive` so it seems like a useless prop that can be removed.

Reply:
Correct!
Removed `isChatSwitcherActive`